src/searchengine/nova3/nova2.py

```diff
@@ -70,18 +70,21 @@ def initialize_engines():
         engi = path.basename(engine).split('.')[0].strip()
         if len(engi) == 0 or engi.startswith('_'):
             continue
-        try:
-            # import engines.[engine]
-            engine_module = importlib.import_module("engines." + engi)
-            # bind class name
-            globals()[engi] = getattr(engine_module, engi)
-            supported_engines.append(engi)
-        except Exception:
-            pass
+        supported_engines.append(engi)
```

Comment:
At least, `supported_engines` should be renamed to something less promising like `found_engines`.

Reply:
done, not tested